concent_api/api_testing_common.py

```diff
@@ -153,15 +152,37 @@ def validate_content_type(actual_content_type: str, expected_content_type: Optio
         )
 
 
+def validate_golem_version(actual_golem_version: str, expected_golem_version: Optional[str]=None) -> None:
```

Comment:
Why is `expected_golem_version` optional? Is there any case where it will be `None`? If not, please make it non-optional.